src/load.ts

```diff
@@ -0,0 +1,75 @@
+import has from './has';
+import Promise from './Promise';
+
+declare var require: Function;
+declare var define: any;
```

Comment:
I'm not sure if we need to provide typings for loader internals. The problem with providing those is that the typings for node's `require` and AMD's `require` are a bit different. Since we're compiling to UMD, most of the need for loader typings is reduced and this module's default function reduce the need for those typings even further.